theme/static/style/style.css

```diff
@@ -98,6 +98,50 @@ ul.topnav li [name="q"] {
     border: 0;
 }
 
+.topnav{
+    overflow: hidden;
+    box-shadow: 0 0.2rem 0.5rem 0 rgba(0,0,0, .3);
+    z-index: 1;
+}
+
+.topnav ul{
+    list-style-type: none;
+    text-align: right;
+}
+
+.topnav li{
+    display: inline-block;
+    vertical-align: top;
+}
+
+.topnav ul a{
+    display: inline-block;
+    padding: 0 1.6rem;
+    text-align: center;
+    line-height: 5rem;
+    border-radius: 0.3rem;
+}
+
+.topnav .search{
+    display: inline-block;
+    height: 50px;
+    padding-left: 2.5rem;
+    border-radius: 0.3rem;
+    border: 0;
+    box-sizing: border-box;
+    font-size: 1.6rem;
+    margin-right: 1.5rem;
+}
+
+.topnav .open-menu{
+    display: none;
+}
+
+.topnav .close-menu{
```

Comment:
the buttons can be selected as `nav .open-menu` or even better `nav button` because there are only 2 of them